Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "interpoli"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+description = ""
+keywords = ["graphics", "animation"]
+categories = ["graphics"]
+repository = "https://github.com/linebender/interpoli"
+readme = "README.md"
+# Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml, with the relevant README.md files
+# and with the MSRV in the `Unreleased` section of CHANGELOG.md.
+rust-version = "1.75"
+
+[lints]
+clippy.doc_markdown = "warn"
+clippy.semicolon_if_nothing_returned = "warn"
+
```

Comment:
Should we have this here? https://github.com/linebender/peniko/blob/3d040a9d4f309c2dbefbd96b64bfb0cc629fec33/Cargo.toml#L43-L111

Reply:
I've added the non-commented out parts and not `clippy.cargo`.